README.md

```diff
@@ -1,9 +1,34 @@
 # Stanford Species Occurrences
+This project provides a Streamlit application that provides record-level verification of individual
+student reports for training and validation purposes as well publishing the records for inclusion
+in [Global Biodiversity Information Facility][GBIF].
 
-## Streamlit Applications
+## Streamlit SPOC Verifier Application
 
-### Abstract Generator
+## Development
+We recommend using a Python virtual environment to install SPOC's package dependencies. SPOC is 
+currently developed with [Python 3.9](https://www.python.org/downloads/) and above. 
 
-### Paper Classifier
+### Setup
+1. Create a new virtual environment `python -m venv spoc-env`
+1. Activate (on POSIX systems) `source spoc-env/bin/activate`
+1. Clone the current SPOC Git repository `git clone https://github.com/sul-dlss-labs/spoc.git`
+1. Enter directory and install dependencies, `cd spoc && pip install -r requirements.txt`  
```

Comment:
Before this I had to run `python3 -m pip install --upgrade pip`

Reply:
👍 just updated to use `python3`